tests/hsc2hs.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# Very stupid fake hsc2hs specific to our tests
+
+awk -v hs="${1%c}" '
```

Comment:
Nice! Is there a way to have this as pure awk script, i.e. not going through `/bin/sh`?